tamr_client/mastering/_mastering.py

```diff
@@ -0,0 +1,83 @@
+from tamr_client import operation
```

Comment:
Can we include a module-level docstring that:
- points to the Mastering docs in docs.tamr.com
- states that the terminology used here is consistent with UI terminology

Reply:
There is a docstring in `mastering/__init__.py` that satisfied part of this:
```python
"""
Tamr - Mastering
See https://docs.tamr.com/docs/overall-workflow-mastering
"""
```
Is it more appropriate for the module-level docstring to live in `_mastering.py`, and if so should there be any docstring in `__init__.py`?

Reply:
I think the duplication is ok since it's only 1-2 lines of docstring duplication. I think contributors will benefit from having the links in both places.